sentry.js

```diff
@@ -1,12 +1,18 @@
-const { sentryDSN } = require("./constants").constants;
+const { sentryDSN, sentryRelease } = require("./constants").constants;
 const sentry = require("@sentry/node");
 
 // Initialize Sentry only if DSN is configured
 if (sentryDSN != null) {
 
-  sentry.init({
-    dsn: sentryDSN,
-  });
+  const sentryConfig = {
+    dsn: sentryDSN
+  };
+
+  if (sentryRelease != undefined) {
```

Comment:
@LearningNerd we should avoid using non-strict equality checks like `!=`. It can produce unintended side effects. I would recommend just doing:
```
  if (sentryRelease) {
```

Reply:
Alrighty, I'll update that line now!